addons/project/static/src/components/project_stop_recurrence_confirmation_dialog/project_stop_recurrence_confirmation_dialog.xml

```diff
@@ -2,14 +2,30 @@
 <templates xml:space="preserve">
 
     <t t-name="project.ProjectStopRecurrenceConfirmationDialog" t-inherit="web.ConfirmationDialog" t-inherit-mode="primary" owl="1">
-        <xpath expr="//button[@t-on-click='_confirm']" position="replace">
-            <button class="btn btn-primary" t-on-click="_confirm">
-                Stop Recurrence
-            </button>
+        <xpath expr="//t[@t-set-slot='footer']" position="before">
+            <div>
+                <p t-if="props.modeLabel == 'delete'">This task is recurrent. Delete it and...</p>
+                <p t-else="">This task is recurrent. Archive it and...</p>
```

Comment:
```suggestion
                <p t-if="props.modeLabel == 'delete'">This task is recurrent. Delete it and...</p>
                <p t-else="">This task is recurrent. Archive it and...</p>
```
We should avoid splitting sentences, as it could create problems with translations.
For instance, in some languages, the verb goes at the end, which would be problematic here, because only the start of the sentence is changed depending on the mode.
As tedious as it seems, I think it would be better to keep the whole sentence together and have one version for each mode.

Reply:
This is what we agreed on with aju. All languages don't put the words in the same order, but I think coordinating conjunctions (here, "and") clearly separate propositions in all languages.
`I am Audric and you are Vincent.`
Possible: `Audric I am and Vincent you are.`
Impossible (right?): `Audric and Vincent I am you are.`

Reply:
That's how all the languages with a "Subject - Object - Verb" structures that I know work, but I've been surprised with weird language oddities in the past (e.g. having a singular + plural version is not enough, because some languages have different plurals for 2-4 and 5+).
Anyway, we don't have to design our interfaces around rare linguistic edge cases that might not happen, so I agree with you, it should be good until someone complains.